### PR TITLE
Split user and admin docs

### LINKED ISSUE #24: Docs
Create user documentation and split it from developer documentation.

### PR DESCRIPTION
**Pull request by SarahAlidoost**
closes #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,47 @@
...
+## For the user
 
-A member of the CO [with "developer" privileges](https://servicedesk.surfsara.nl/wiki/display/WIKI/Appoint+a+CO-member+a+developer) can register the current application as a Catalog Item on SURF Research Cloud following [these instructions](https://servicedesk.surf.nl/wiki/display/WIKI/Create+your+own+catalog+items).
+### Access the existing workspace
 
-The following steps create a catalog item with the following three paraters:
-* the number of worker nodes in the cluster of VMs;
-* (optional) the URL to a conda environment file, employed to configure both the head node and the workers with the same Python environment;
-* (optional) a macaroon to authenticate access to [the SURF dCache starage](http://doc.grid.surfsara.nl/en/latest/Pages/Advanced/storage_clients/webdav.html#sharing-data-with-macaroons) via [dCacheFS](https://github.com/NLeSC-GO-common-infrastructure/dcachefs).
+To access the existing workspace, first ask administrators to invite you. After
```
Comment (fnattino) [inline README.md:11]:
```suggestion
To access an existing workspace, first ask administrators to invite you to a collaborative organization (CO). After
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,47 @@
...
+receiving the invitation link, you will be directed to the [log in
+page](https://portal.live.surfresearchcloud.nl/). Follow the instructions in
+[Log in to your
+workspace](https://servicedesk.surf.nl/wiki/display/WIKI/Log+in+to+your+workspace)
+to access your workspace.
+
+### Create a new workspace
+
+See the instructions [For the administrator](#for-the-administrator) to create a
```
Comment (fnattino) [inline README.md:20]:
```suggestion
See the instructions ["Create the workspace"](#3-create-the-workspace) below to create a
```

File: README.md [hunk 3]
```diff
@@ -1,19 +1,47 @@
...
+[request
+portal](https://www.surf.nl/en/research-it/apply-for-access-to-compute-services).
+When your application is approved, SURF will send you an invitation with the
+login to SRC. To get familiar with the SRC, have a look at the [SRC First
+Steps](https://servicedesk.surf.nl/wiki/display/WIKI/First+Steps).
+
+A member of the CO [with "developer"
+privileges](https://servicedesk.surfsara.nl/wiki/display/WIKI/Appoint+a+CO-member+a+developer)
+can register the current application as a `Catalog Item` and setting up these
+three parameters:
```
Comment (fnattino) [inline README.md:35]:
```suggestion
can register the current application as a `Catalog Item` (steps 1. and 2. below). Once created, the catalog item will allow him/her as well as other members of the 
CO to create and access a workspace with JupyterHub and Dask (step 3. below). When creating a workspace, users will be
able to set three parameters:
```

File: README.md [hunk 4]
```diff
@@ -24,29 +52,65 @@ instructions, using the following parameters:
...
+    "Default parameter value" to `default`.
+
+### 3. Create the workspace
+
+To create a workspace, you need to first create a storage, see [External storage
+volumes](https://servicedesk.surf.nl/wiki/display/WIKI/External+storage+volumes).
+Then, you can create a new workspace by following [Start a simple
+workspace](https://servicedesk.surf.nl/wiki/display/WIKI/Start+a+simple+workspace)
+and attach your storage to it. For the application, select the catalog item
+created in the previous steps.
```
Comment (fnattino) [inline README.md:97]: Actually, for I am not sure whether the storage will work with this component (the workspace includes multiple VMs, and I think a storage can be bound to a single VM only. To be tested, will raise an issue about this! 

```suggestion
You can create a new workspace by following [Start a simple
workspace](https://servicedesk.surf.nl/wiki/display/WIKI/Start+a+simple+workspace). 
For the application, select the catalog item
created in the previous steps. Follow the instructions and select parameters/provide input 
when requested.
```

**Review by fnattino — CHANGES_REQUESTED** [review]
Thanks a lot for cleaning this up so thoroughly @SarahAlidoost, this reads very clear to me. I have added some suggestions for minor improvements. The only thing that should be removed is the point on storage, which I think is not supported by this component.  Feel free to merge afterwards!

**Comment by SarahAlidoost** [comment]
> Thanks a lot for cleaning this up so thoroughly @SarahAlidoost, this reads very clear to me. I have added some suggestions for minor improvements. The only thing that should be removed is the point on storage, which I think is not supported by this component. Feel free to merge afterwards!

Thanks for reviewing and commenting on external storage. I thought this storage is bound to the main VM that runs the notebook and other VMs can write on it. good to test it :+1:

**Review by fnattino — APPROVED** [review]
Perfect, thanks a lot!

**Pull request merged**